### DISCUSSION TITLE
Account Flagged for a Month- No Response from Support

### DISCUSSION THREAD

**Discussion by ofcsayan** [community/community#126064] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

It's been over a month since my  GitHub account got flagged, and despite reaching out to support, I haven't received any response.
Upon checking the GitHub Security log, I noticed login activity from an unfamiliar country, indicating a possible hack. I promptly changed the password, but the account remains flagged.
Support is not responding. What should be our next steps? Any advice would be greatly appreciated.

Account: @sayann70
Ticket ID: #2704312

**Comment by queenofcorgis** [comment, marked as answer]
Hi @ofcsayan, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
